sherlock_test.py

```diff
@@ -0,0 +1,151 @@
+                                    tor=tor,
+                                    unique_tor=unique_tor
+                                   )
+        for social_network, result in results.items():
+            if result['exists'] == "no":
+                 logging.info(f"Username '{username}' not detected "
+                              f"on {social_network}."
+                             )
+            else:
+                 #At least one test failed.
+                 tests_succeeded = False
+                 logging.error(f"Username '{username}' detected "
+                               f"on {social_network}."
+                              )
+
+    return tests_succeeded
+
+
+def main():
+    # Colorama module's initialization.
+    init(autoreset=True)
+
+    version_string = f"%(prog)s {__version__}\n" +  \
+                     f"{requests.__description__}:  {requests.__version__}\n" + \
+                     f"Python:  {platform.python_version()}"
+
+    parser = ArgumentParser(formatter_class=RawDescriptionHelpFormatter,
+                            description=f"{module_name} (Version {__version__})"
+                           )
+    parser.add_argument("--version",
+                        action="version",  version=version_string,
+                        help="Display version information and dependencies."
+                       )
+    parser.add_argument("--verbose", "-v", "-d", "--debug",
+                        action="store_true",  dest="verbose", default=False,
+                        help="Display extra debugging information."
+                       )
+    parser.add_argument("--quiet", "-q",
+                        action="store_false", dest="verbose",
+                        help="Disable debugging information (Default Option)."
+                       )
+    parser.add_argument("--tor", "-t",
+                        action="store_true", dest="tor", default=False,
+                        help="Make requests over TOR; increases runtime; requires TOR to be installed and in system path.")
+    parser.add_argument("--unique-tor", "-u",
+                        action="store_true", dest="unique_tor", default=False,
+                        help="Make requests over TOR with new TOR circuit after each request; increases runtime; requires TOR to be installed and in system path.")
+
+    args = parser.parse_args()
+
+    # Load the data
+    data_file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data.json")
+    with open(data_file_path, "r", encoding="utf-8") as raw:
+        site_data_all = json.load(raw)
+
+
+    #Setup logging to show informational messages.
+    logging.basicConfig()
+    logging.getLogger().setLevel(logging.INFO)
+
+    result_summary = []
+    site_data = {site: site_data_all[site]
+                       for site in ['Twitter', 'eBay'] if site in site_data_all
+                }
+    result_summary.append(sherlock_test(['hoadlck'],
```

Comment:
Probably making a JSON file for exists and not exists usernames for each social network to use in test would be a good idea.

Reply:
Also for exists usernames if we used the ones that we sure never changes is way more better. like using username of the founder for some social networks, like `zuck` for facebook or `jack` for twitter.